CHANGELOG.md

```diff
@@ -14,6 +14,28 @@ The Azure MCP Server updates automatically by default whenever a new release com
 
 - Added caching for Cosmos DB databases and containers. [[813](https://github.com/Azure/azure-mcp/pull/813)]
 
+#### Dependency Updates
+
+- Upgraded `Azure.Core` from `1.46.2` to version `1.47.1`.
```

Comment:
Maybe say: "updated the following dependencies to improve .NET Ahead-of-time compilation (AOT) support. AOT will enable shipping Azure MCP Server as a self-contained EXE"